Formula/source-to-image.rb

```diff
@@ -0,0 +1,20 @@
+class SourceToImage < Formula
+  desc "Tool for building source and injecting into docker images"
+  homepage "https://github.com/openshift/source-to-image"
+  url "https://github.com/openshift/source-to-image.git",
+    :tag => "v1.1.3",
+    :revision => "ddb10f16412977688136b77c76c30c0632e510d6"
+  head "https://github.com/openshift/source-to-image.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "hack/build-go.sh"
+    bin.install "_output/local/bin/darwin/amd64/s2i"
+  end
+
+  test do
+    system "#{bin}/s2i", "create", "testimage", testpath
+    assert (testpath/"Dockerfile").exist?
```

Comment:
Can you add a second argument here `, "Dockerfile does not exist!"` as the default message is pretty terrible. Thanks!